src/lib/calculation.ts

```diff
@@ -6,7 +6,7 @@ import {
   parseInputNumber,
 } from './utils'
 import Decimal from 'decimal.js'
-import { sum } from './utils'
+import { sum, ceilDecimal } from './utils';
```

Comment:
bezi ti prettier? vyzera ze `;` je extra

Reply:
Nejako sa to neautofixlo

Reply:
Mozno idea na github action: kazdy PR prejst prettierom a fixnut/commitnut automaticky. Aj ked taky autocommit moze byt risky, ale asi by to stalo za to

Reply:
A automaticky vyhodit `.only` na testoch a odkomentovat useCasy v `cases`, lebo to vzdy zabudam :D

Reply:
https://github.com/priznanie-digital/priznanie-digital/pull/177

Reply:
#177 sa mi paci :) Lepsie ako automaticky commit podla mna.